utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java

```diff
@@ -654,6 +660,111 @@ String dsmExtract(
         return "ok";
     }
 
+    @RequestMapping("/dsm/manual/produce")
+    String dsmManualCheckpointProduce(
```

Comment:
Nit: you can avoid having `HttpServletResponse` as a parameter and just use `ResponseEntity<String>` as return type (https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html)

Reply:
Thanks!